lib/Doctrine/ORM/Tools/Setup.php

```diff
@@ -121,6 +124,12 @@ public static function createXMLMetadataConfiguration(array $paths, $isDevMode =
      */
     public static function createYAMLMetadataConfiguration(array $paths, $isDevMode = false, $proxyDir = null, ?Cache $cache = null)
     {
+        Deprecation::trigger(
+            'doctrine/orm',
+            'https://github.com/doctrine/orm/issues/8465',
+            'YAML mapping driver is deprecated and will be removed in Doctrine ORM 3.0, please migrate to annotation or XML driver.'
```

Comment:
```suggestion
            'YAML mapping driver is deprecated and will be removed in Doctrine ORM 3.0, please migrate to attribute or XML driver.'
```

Reply:
This message is C&P. Shall I adjust the other occurences as well?

Reply:
That would be great 👍